File: count_words.py
import re

# Read the file
with open('static/words_filtered.js', 'r', encoding='utf-8') as f:
    content = f.read()

# Extract all words between quotes
words = re.findall(r'"([A-Z]+)"', content)

print("=== TUSMO CESI - Word Count ===\n")
print(f"Total words: {len(words)}")

# Count by length
by_length = {}
for word in words:
    length = len(word)
    by_length[length] = by_length.get(length, 0) + 1

print("\nWords by length:")
for length in sorted(by_length.keys()):
    print(f"  {length} letters: {by_length[length]} words")

print("\n=== End ===")

File: MotMystere/count_words.js
// Count words in the database
import { WORDS } from "./static/words_filtered.js";

console.log("=== TUSMO CESI - Word Count ===\n");

// Total count
console.log(`Total words: ${WORDS.length}`);

// Count by length
const byLength = {};
WORDS.forEach(word => {
    const len = word.length;
    byLength[len] = (byLength[len] || 0) + 1;
});

console.log("\nWords by length:");
for (let len = 5; len <= 10; len++) {
    console.log(`  ${len} letters: ${byLength[len] || 0} words`);
}

console.log("\n=== End ===");
